requirements.txt

```diff
@@ -12,6 +12,7 @@ django-cors-headers==2.4.0
 django-environ==0.4.5
 django-helusers==0.4.2
 django==2.1.3
+djangorestframework-gis==0.14
```

Comment:
You are not using this yet, right? 
Same functionality could be done with [`munigeo`](https://github.com/City-of-Helsinki/berth-reservations/blob/master/harbors/api.py#L4) btw, but this is good too, I think. 🙂